src/reducers/ReducerWeather.js

```diff
@@ -0,0 +1,9 @@
+import { FETCH_WEATHER } from '../actions/index';
+
+export default function(state = [], action) {
+    switch(action.type) {
+        case FETCH_WEATHER:
+            return [ action.payload.data, ...state ];
```

Comment:
Please add a comment with a dummy value of payload data, which shows the content structure of `action.payload.data`